.circleci/config.yml

```
@@ -104,6 +104,7 @@ workflows:
             branches:
               only:
                 - develop
+                - release
```

Need to handle dates in the release branch name

```suggestion
                - /^release-.*/
```

---

(discussed on slack)